web/html/src/manager/admin/config/monitoring.js

```diff
@@ -0,0 +1,250 @@
+            if (allSuccess) {
+              newStatus = true;
+              msg = MessagesUtils.success(msgMap["enabling_succeeded"]);
+            } else if (someFailed) {
+              msg = MessagesUtils.error(msgMap["enabling_failed_partially"]);
+            } else {
+              msg = MessagesUtils.error(msgMap["enabling_failed"]);
+            }
+          } else { // disabled -> disabled, enabled -> enabled
+            msg = MessagesUtils.info(msgMap["no_change"]);
+          }
+          this.setState({
+            messages: msg,
+            exportersStatus: nullify(this.state.exportersStatus),
+            monitoringEnabled: newStatus,
+            loading: false,
+            error: false
+          });
+
+        } else {
+          this.setState({
+            messages: MessagesUtils.error(msgMap["unknown_status"]),
+            exportersStatus: nullify(this.state.exportersStatus),
+            loading: false,
+            error: false
+          });          
+        }
+        this.getMonitoringStatus();
+      })
+      .catch(this.handleResponseError);      
+    }
+  }
+
+  handleResponseError = (jqXHR: Object, arg: string = '') => {
+    const msg = Network.responseErrorMessage(jqXHR,
+      (status, msg) => msgMap[msg] ? t(msgMap[msg], arg) : null);
+    this.setState({messages: msg, loading: false, error: true});
+  }
+
+  iconType = (value) => {
+    if (value === true) {
+      return "item-enabled";
+    } else if (value === false){
+      return "item-error";
+    } else {
+      return "item-disabled";
+    }
+  }
+  
+  render() {
+      const keys = Object.keys(this.state.exportersStatus).sort();
+
+      const exporters = <ul style={{listStyle: 'none', paddingLeft: '0px'}}>{
+        keys.map(key => 
+          this.state.exportersStatus ? 
+          (<li>
+            <Icon type={this.iconType(this.state.exportersStatus[key])} className="fa-1-5x"/>
+            { key in exporterMap ? exporterMap[key] : capitalize(key)}
+            </li>)
+          : null
+          )
+      }
+      </ul>
+
+      return (
+      <div className="responsive-wizard">
+        <Messages items={this.state.messages}/>
+        <div className="spacewalk-toolbar-h1">
+          <div className="spacewalk-toolbar"></div>
+          <h1>
+            <i className="fa fa-info-circle"></i>
+            SUSE Manager Configuration - Monitoring
+            <a href="/rhn/help/reference/en-US/ref.webui.admin.config.jsp#s3-sattools-config-gen" target="_blank">
+              <i className="fa fa-question-circle spacewalk-help-link"></i>
+            </a>
+          </h1>
+        </div>
+        <div className="page-summary">
+          <p>
+            {t("Setup your SUSE Manager server monitoring.")}
+          </p>
+        </div>
+        <div className='spacewalk-content-nav'>
+          <ul className="nav nav-tabs">
```

Comment:
There's already a component that does something similar (`<TabContainer>`) but it works only when all tabs are react components. It would need to be extended to allow mixing old-style pages and React components. But I think that should be done in a separate PR.